collateral-info.yaml

```diff
@@ -215,6 +215,10 @@ collateral-defs:
     collateral-shortname: 6s Capital
     collateral-symbol: RWA001
     collateral-info-url: https://manual.makerdao.com/collateral/real-world-assets-in-detail/6s
+    collateral-stability-fee-type: Fixed
+    collateral-stability-fee-payment-frequency: By the second
```

Comment:
Suggest 'By the second' -> 'Per second' and using this convention on all `collateral-stability-fee-payment-frequency` values. Per Month etc. One-time should remain an exception.

Reply:
done